Open Judge System/Web/OJS.Web/Areas/Contests/Views/Submissions/View.cshtml

```diff
@@ -100,6 +83,22 @@
 }
 else
 {
+    if (Model.UserHasAdminPermission)
+    {
+        <div class="clearfix"></div>
+        if (!string.IsNullOrWhiteSpace(Model.ProcessingComment))
+        {
+            <h2>@Resource.Execution_result:</h2>
+            <pre>@Model.ProcessingComment</pre>
+            if (Model.UserHasAdminPermission)
+            {
+                <pre>@Resource.Exception_type: @Model.ExceptionType  
+            @Model.ExecutionComment
```

Comment:
The spacing seems a bit off here. Row 99 is unnesessary.

Reply:
Fixed